### PR TITLE
Device Frontend

### PR DESCRIPTION
**Pull request by t-harps**
1. Improved the checkin table to show relevent parameters.
2. Changed wording on dashboard to use pluralize devices/developers.
3. Removed unecessary methods from checkin model as a result
4. Made the delete link on a checkin work correctly.

### CODE REVIEW

File: app/views/users/devices/_checkin_table.html.erb [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+        <%= checkin.lng %>
+      </td>
+      <td>
+        <%= checkin.country_code %>
+      </td>
+      <td>
+        <%= checkin.created_at %>
+      </td>
+      <td>
+        <%= button_to "x", checkin_user_device_path(current_user.id, @device.id, checkin_id: checkin.id), method: :delete, class: "btn btn-danger" %>
```
Comment (dansteele) [inline app/views/users/devices/_checkin_table.html.erb]: This is no longer using Ajax; does this still have the desired effect?

Comment (t-harps) [inline app/views/users/devices/_checkin_table.html.erb]: Previously the page wasn't being updated when this button was clicked, you had to refresh the page to see the Checkin was deleted. With the Ajax removed it seems to update properly.

Comment (dansteele) [inline app/views/users/devices/_checkin_table.html.erb]: Aight :)
This is the file that was previously returned. Either re-implement it or remove the file :)
https://github.com/earlymarket/CoPosition/blob/master/app/views/users/devices/checkin.js.erb

Comment (dansteele) [inline app/views/users/devices/_checkin_table.html.erb]: Not sure at what point the functionality was lost

Comment (t-harps) [inline app/views/users/devices/_checkin_table.html.erb]: At some point a redirect was added to the Checkin method in the devices controller, i believe this was causing the javascript to be ignored, put Ajax back in and works as it should now I believe.

**Pull request merged**